static/js/utils/getHHConfigAttributes.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Returns an object which contains the data-hhconfig attributes on the element specified by the selector
+ * 
+ * @example
+ * If the attributes on the element contain 'data-hhconfig-link-target="_top"', the returned
+ * object will be { link-target: "_top" }
+ *
+ * @param {string} selector
+ * @returns {Object}
+ */
+export default function getHHConfigAttributes (selector) {
+  const element = document.querySelector(selector);
+  if (!element) {
+    return {};
+  }
+  const attributes = element.attributes;
```

Comment:
I think it'd be simpler to access the data-* attributes directly (`element.dataset`)

Reply:
I was hesitant to use that due to the automatic conversion to camelcase which I thought might make complicate things. I'll get product's input on it

Reply:
ah that's fair, feel free to ack this